.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true
+}
```

Comment:
Actually I want to add parantheses if there are more than one parameter, but it can be added anyway.

```es6
// more than one parameter
new Promise((resolve, reject) => {
    if (consumer instanceof RestClient) {}
}

// one parameter
.then(response => {});
```